src/filter.rs

```diff
@@ -67,7 +56,18 @@ impl Filter {
                 .any(|ext| loc.file().ends_with(&format!(".{}", ext)))
     }
 
-    pub fn new (matches: &getopts::Matches) -> Filter {
+    /// An abbreviation for the filter checks.
```

Comment:
The commit message should probably say which function was turned into a method..